addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -123,6 +124,18 @@ var QuestionFormWidget = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Handler when a user clicks on the 'Save & New' button.
+     * @param ev
+     * @private
+     */
+    _validateAndCreateNewQuestion: function () {
```

Comment:
Better to update the docstring of `_validateQuestion`.
```suggestion
    _validateAndCreateNewQuestion() {
```

Reply:
I gave it a try, but it does not work.